docs/pages/billing/overview.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: 'Billing: Overview'
+sidebar_title: Overview
+description: An overview of information on billing and subscriptions to manage your EAS account's plans, invoices, receipts, payments, and usage.
+---
+
+import { BoxLink } from '~/ui/components/BoxLink';
+import { CreditCard02Icon } from '@expo/styleguide-icons';
+
+Expo provides various subscription plans for integrated cloud services through Expo Application Services (EAS). You can manage and track invoices, payments, plans, and other billing-related information on the **Billing and Receipts** pages in your account's dashboard. Only account Owners and Admins have access to this page.
+
+See our list of resources below to learn more about different aspects of billing and subscriptions:
+
+## Plans
+
+<BoxLink
+  title="Subscriptions, plans, and add-ons"
+  description="In-depth guide on available Expo Application Services (EAS) plans and how they work, usage-based pricing, and add-ons."
+  href="/billing/plans/"
+  Icon={CreditCard02Icon}
+/>
+
+<BoxLink
+  title="Manage plans"
+  description="Learn how to update, downgrade, or cancel your Expo account's plan."
+  href="/billing/manage/#manage-plans"
+  Icon={CreditCard02Icon}
+/>
+
+## Manage billing
+
+<BoxLink
+  title="Manage billing"
+  description="Learn how manage billing information of your Expo account."
+  href="/billing/manage/#manage-billing-information"
+  Icon={CreditCard02Icon}
+/>
+
+<BoxLink
+  title="Payments, invoices, and receipts"
+  description="Learn how to view your account's payment history, download invoices and receipts, and request a refund for a charge."
+  href="/billing/invoices-and-receipts/"
+  Icon={CreditCard02Icon}
+/>
+
+## Usage-based pricing
+
+<BoxLink
+  title="Usage-based pricing"
+  description="Learn how Expo applies usage-based billing for customers who exceed their plan limits and how to monitor your EAS Build usage."
```

Comment:
```suggestion
  description="Learn how Expo applies usage-based billing for customers who exceed their plan quotas and how to monitor your EAS Build usage."
```
Not 100% sure this is the right word, but wanted to avoid the implication that there's "limits". There's just built in usage and usage beyond that.

Reply:
You are right. We don't use that term. I think your suggestion fits well here.